### PR TITLE
ANDROID-10011 Set max lines to CardView title and description

### PR DESCRIPTION
**Pull request by pmartinbTEF**
### :goal_net: What's the goal?
Add max lines parameter for title and description in CardView

### :construction: How do we do it?
Added cardTitleMaxLines and cardDescriptionMaxLines atributes

### ☑️ Checks
- [ ] I updated the corresponding readmes
- [ ] Tested in Smart WiFi with this tutorial: https://github.com/Telefonica/android-messenger/wiki/Testing-libraries

### :test_tube: How can I test this?
Before
drawing

After
drawing

### CODE REVIEW

File: library/src/main/java/com/telefonica/mistica/card/CardView.kt [hunk 1]
```diff
@@ -32,11 +34,21 @@ import com.telefonica.mistica.util.getThemeColor
         attribute = "cardTitle",
         method = "setTitle"
     ),
+    BindingMethod(
+        type = MediaCardView::class,
```
Comment (jeslat) [inline library/src/main/java/com/telefonica/mistica/card/CardView.kt:38]: I think the type should be `CardView::class`, shouldn't be?

File: library/src/main/java/com/telefonica/mistica/card/CardView.kt [hunk 2]
```diff
@@ -32,11 +34,21 @@ import com.telefonica.mistica.util.getThemeColor
...
+        attribute = "cardTitleMaxLines",
+        method = "setTitleMaxLines"
+    ),
     BindingMethod(
         type = CardView::class,
         attribute = "cardDescription",
         method = "setDescription"
     ),
+    BindingMethod(
+        type = MediaCardView::class,
```
Comment (jeslat) [inline library/src/main/java/com/telefonica/mistica/card/CardView.kt:48]: Same here

**Review by yamal-alm — APPROVED** [review]
nice!

**Review by gmerinojimenez — APPROVED** [review]
Nice work!

File: library/src/main/java/com/telefonica/mistica/card/CardView.kt (hunk 2 above)
Comment (pmartinbTEF) [reply library/src/main/java/com/telefonica/mistica/card/CardView.kt]: Done

File: library/src/main/java/com/telefonica/mistica/card/CardView.kt (hunk 1 above)
Comment (pmartinbTEF) [reply library/src/main/java/com/telefonica/mistica/card/CardView.kt]: Done

**Review by jeslat — APPROVED** [review]
good job!

**Pull request merged**